src/blockchains/ethereum/converter_events.ts

```diff
@@ -0,0 +1,161 @@
+import Web3 from 'web3';
+import { ERC20Token } from './abis';
+import { fromWei, toRatio } from '../../helpers';
+import { ConversionEvent, TokenRateEvent } from '../../types';
+
+const GENESIS_BLOCK_NUMBER = 3851136;
+
+const OWNER_UPDATE_EVENT_HASH = Web3.utils.keccak256("OwnerUpdate(address,address)");
+
+const CONVERSION_EVENT_LEGACY = [
```

Comment:
why not put these in the abis file?

Reply:
Because we don't have the source code from which they are generated (while the ABIs are pretty much taken directly from the compilation output).

Reply:
so? still abis regardless of where they came from

Reply:
done